### PR TITLE
team page fixes (#35, #40, #46)

### PR DESCRIPTION
**Pull request by rpietraszko**
## TO DO:
* [X] add workaround for safari browser, since `load()` behavior created
  weird visible black screen for 1 sec (https://trello.com/c/tkdm6rbU/40-improve-gifs-performance)
* [X] update nginx config to add expire head for mp4 and webm files
* [X] do not show videos below tablet breakpoint (https://trello.com/c/G4goM79d/46-team-page-iphone-play-button-bug)
* [X] set correct placeholder for the last join-us element, fix his
  positioning issues (https://trello.com/c/ogntau14/35-team-page-picture-reverses-join-us-circle)

---

Review:
* [ ] @MagdaMalinowska
* [ ] @aberracja
* [ ] @kruszczynski
* [x] @pjar
* [x] @AdamMankowski

### CODE REVIEW

**Comment by AdamMankowski** [comment]
Hover does not work on Safari :(

**Comment by rpietraszko** [comment]
> Hover does not work on Safari

I know, it's super weird - but it does not load videos only on localhost. Try to run project on docker or just ship it on staging and it will work all right.

File: source/stylesheets/modules/_m-team.scss [hunk 1]
```diff
@@ -117,6 +145,10 @@
       @include font(22px, 30px);
       margin-top: 10px;
     }
+
+    .no-content {
+      width: 20vw;
```
Comment (pjar) [inline source/stylesheets/modules/_m-team.scss]: Seems odd that this has to be set for every media query.

Comment (rpietraszko) [inline source/stylesheets/modules/_m-team.scss]: Any other ideas? We need it to align last `join-us` element properly with those filler grid elements (bug described in https://trello.com/c/ogntau14/35-team-page-picture-reverses-join-us-circle)

**Pull request merged**